examples/basic_use.py

```diff
@@ -42,6 +40,7 @@
 
 # Or modify the List of existing labels. The new label is unicode with no
 # spaces
+assert issue.fields.labels  # help mypy
```

Comment:
just for my interest what do you mean with help mypy?

Reply:
So mypy thinks that the `issue.fields.labels` can be `None`, but it has certain ways to infer the type of a variable. This can be via `cast()`, `isinstance()` or `assert` statements - refer: https://mypy.readthedocs.io/en/stable/casts.html